### ISSUE TITLE
Only single updated reaction visible on Teams

### ISSUE THREAD

**Issue by AayushChaudhary0001** [mattermost/mattermost-plugin-msteams-sync#23]
Despite of multiple reactions added by the user on any desired message on MM, only the last added reaction is visible on that particular message on Teams.

**Steps to reproduce:-**
1. Add any desired reaction to the message in MM.
2. Add second desired reaction to the same message in MM.
3. Verify the same reactions on Teams.

Only the last added reaction would be presented to the user on Teams.

**Comment by jespino** [comment]
I don't think this is a bug. AFAIK MS Teams only supports 1 reaction  per user/per post. If you add a reaction to a post in MS Teams, and then you add another one, it removes the original one and replace it with the new one.

**Comment by manojmalik20** [comment]
Closing this issue as it's not valid.
